GameServerLib/Packets/PacketHandlers/HandleCastSpell.cs

```diff
@@ -35,6 +35,15 @@ public override bool HandlePacket(int userId, CastSpellRequest req)
                 return false;
             }
 
+            if (s.CastInfo.SpellSlot == (int)SpellSlotType.InventorySlots)
+            {
+                var item = s.CastInfo.Owner.Inventory.GetItem(s.SpellName);
+                if (item.ItemData.ItemClass == 1)
```

Comment:
`ItemClass` variable does not reflect consumability.
Use `Consumed` variable instead.

Reply:
resolved